lib/binding_rust/ffi.rs

```diff
@@ -9,6 +9,7 @@ include!(concat!(env!("OUT_DIR"), "/bindings.rs"));
 include!("./bindings.rs");
 
 #[cfg(any(unix, target_os = "wasi"))]
+#[cfg(feature = "std")]
```

Comment:
Should be merged with the above condition and also added to the windows version.